apps/expo/src/components/AnnouncementTile.tsx

```diff
@@ -0,0 +1,55 @@
+import { Image, Text, View } from "react-native";
+import type { ImageSourcePropType } from "react-native";
+
+interface Props {
+  title: string;
+  titleImageSrc: ImageSourcePropType;
+  createdAt: Date;
+  createdBy: string;
+  userImageSrc: ImageSourcePropType;
+}
```

Comment:
Use the RouterOutputs helper in utils/api.tsx to infer the type of an Announcement from the getAnnouncements endpoint

Reply:
I'm not sure how to use it because the getAnnouncement endpoint has different types than what I am using. For example, I need createdBy to be a user name, but the endpoint only has the userID for the corresponding user, which is a number. Is there a way to go about this in TypeScript?